### PR TITLE
Calendar work

### PR DESCRIPTION
**Pull request by S00185883**
dont worry the green wont stay like that
Screenshot 2021-08-11 at 10 08 43

### CODE REVIEW

File: src/components/calendar/Calendar.js [hunk 1]
```diff
@@ -40,14 +51,17 @@ function Calendar({ setParentState, parentState, setDateSelected }) {
...
+          //start: new Date(),
         }}
         fixedWeekCount={false}
         eventColor="#378006"
         backgroundColor="#000565"
         events={theEvents}
       />
-    </Col>
+      
+    </Col></div>
```
Comment (Clancinio) [inline src/components/calendar/Calendar.js:62]: Looks like your formatter is not working?

File: src/components/calendar/Calendar.js [hunk 2]
```diff
@@ -20,12 +20,11 @@ function Calendar({ setParentState, parentState }) {
       
       setEvents( theResponse );
       console.log( theEvents );
+
       theResponse.forEach(element => {
         const background = element.color;
       let root = document.documentElement;
       root.style.setProperty( '--background-color', background );
```
Comment (Clancinio) [inline src/components/calendar/Calendar.js]: Make sure you have format on save selected :)

Comment (S00185883) [reply src/components/calendar/Calendar.js]: i do. thats how it formatted it but it looks weird to me too

Comment (Clancinio) [reply src/components/calendar/Calendar.js]: I think there is closing brackets you don't need that's why it's weird

File: src/components/calendar/Calendar.js [hunk 3]
```diff
@@ -20,12 +20,11 @@ function Calendar({ setParentState, parentState }) {
...
-        console.log( background );
-        console.log( element.color );
       });
      
     } );
```
Comment (Clancinio) [inline src/components/calendar/Calendar.js:32]: These needed?

File: src/components/calendar/Calendar.js [hunk 4]
```diff
@@ -16,19 +16,30 @@ function Calendar({ setParentState, parentState, setDateSelected }) {
...
+      setEvents( theResponse );
+      console.log( theEvents );
 
+      theResponse.forEach(element => {
+        const background = element.color;
+      let root = document.documentElement;
+      root.style.setProperty( '--background-color', background );
+      });
+     
+    } );
```
Comment (Clancinio) [inline src/components/calendar/Calendar.js:32]: Think these can be removed - should solve formatting issues

**Review by Clancinio — APPROVED** [review]

**Pull request merged**